README.md

```diff
@@ -19,7 +19,7 @@ Quick links:
 To install Go, visit [this link](https://golang.org/dl/).
 
 ### Installing MSAL Go
-`go get -u github.com/AzureAD/microsoft-authentication-library-for-go/`
+`go get -u github.com/AzureAD/microsoft-authentication-library-for-go/msal`
```

Comment:
Since you're updating this line can you please remove the `-u`?  This will trigger upgrading your dependencies in `go.mod` which is likely wrong (we had a customer complain about this in our Event Hubs repo).

Reply:
Yes I will make that change